src/e2e-test/java/io/cdap/plugin/utils/PubSubClient.java

```diff
@@ -56,4 +104,186 @@ public static String getTopicCmekKey(String topicId) throws IOException {
     return getTopic(topicId).getKmsKeyName();
   }
 
+  public static void publishWithErrorHandlerExample(String projectId, String topicId, List<String> jsonMessages)
+    throws IOException, InterruptedException {
+    TopicName topicName = TopicName.of(PluginPropertyUtils.pluginProp(ConstantsUtil.PROJECT_ID), topicId);
+    Publisher publisher = null;
+    try {
+      publisher = Publisher.newBuilder(topicName).build();
+
+      for (final String message : jsonMessages) {
```

Comment:
change the jsonMessage to something relevant

Reply:
changed to messages.